src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const stylesObject = {};
+
+  const lines = sourceString.split(';');
+
+  lines.forEach(function (line) {
+    const part = line.split(':');
+
+    if (part.length >= 2) {
+      stylesObject[part[0].trim()] = part[1].trim();
+    }
+    
+  });
```

Comment:
Consider using more functional approaches like `map` and `filter` instead of `forEach` for creating a new object. This would align with the checklist item of avoiding loops and using iteration methods.